webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/ldap/LDAPEntityMapper.scala

```diff
@@ -1142,7 +1142,7 @@ class LDAPEntityMapper(
     mod.resetValuesTo(A_API_UUID, principal.id.value)
     mod.resetValuesTo(A_NAME, principal.name.value)
     mod.resetValuesTo(A_CREATION_DATETIME, GeneralizedTime(principal.creationDate).toString)
-    mod.resetValuesTo(A_API_TOKEN, principal.token.exposeHash())
+    mod.resetValuesTo(A_API_TOKEN, principal.token.exposeHash().getOrElse(""))
```

Comment:
I don't think that will work. The API token is a mandatory field in the schema for `apiAccount`. 
Perhaps we want to be able to have that case, but it seems to be a hard hypothesis in the lower layout. We can perhaps workaround it with a marker value that would be extracted as none.

Reply:
or even `v2:`